### PR TITLE
DOC: Preliminary edit of 1.13.0 release notes.

### PR DESCRIPTION
**Pull request by charris**
[ci skip]

### CODE REVIEW

**Comment by charris** [comment]
This is a first pass. I'd like it in now so that anything new that goes in will conform.

Hmm, it seems that neither `PyArray_MapIterArrayCopyIfOverlap` nor older functions such as `PyArray_MapIterArray` are documented in the official documentation. Anyone know if there is other documentation out there? We really need to make sure the C-API is documented.

File: doc/release/1.13.0-notes.rst [hunk 1]
```diff
@@ -29,9 +32,8 @@ Future Changes
 Compatibility notes
 ===================
 
-
 Error type changes
-~~~~~~~~~~~~~~~~~~
```
Comment (eric-wieser) [inline doc/release/1.13.0-notes.rst]: 1.11, 1.10, 1.9, 1.8, and probably more, all used tildes for these headings. 1.12 is the outlier, not 1.13

**Comment by rgommers** [comment]
> Anyone know if there is other documentation out there? 

Don't think so. There are more issues about things missing from the C API docs.

> We really need to make sure the C-API is documented.

What we need is an automated way to check that every public function (Python and C) is present in the reference guide. Otherwise we'll keep missing this. We have this for scipy [here](https://github.com/scipy/scipy/blob/master/.travis.yml#L21), would be good to copy that and extend for the C API.

File: doc/release/1.13.0-notes.rst (hunk 1 above)
Comment (charris) [inline doc/release/1.13.0-notes.rst]: It's not dashes the outlier, it's dashes, a new hope.

I introduced tildes back when to solve one rendering problem, and now that we are releasing on github, dashes for subtitles and stars for lists gives us better markdown compatibilit.

Comment (eric-wieser) [inline doc/release/1.13.0-notes.rst]: Note that we have a regression in #8845 here, which was merged at just the wrong time

**Pull request merged**